### PR TITLE
add some spells

### PR DESCRIPTION
**Pull request by h-ohta**
`XYZIRADT` = X, Y, Z, Intensity, Ring, Azimuth, Distance and Timesramp

### CODE REVIEW

File: cspell/.cspell.json [hunk 1]
```diff
@@ -985,6 +986,7 @@
         "unsqueeze",
         "URDF",
         "Uref",
+        "Urefex",
```
Comment (kenji-miyake) [inline cspell/.cspell.json:989]: I think `transed` and `Urefex` shouldn't be added because they can be replaced by `transformed` and `Uref_ex`.

Comment (h-ohta) [reply cspell/.cspell.json:989]: ok, i'll fix the codes which is using them instead

File: cspell/.cspell.json [hunk 2]
```diff
@@ -297,6 +297,7 @@
         "FASTCRC",
         "favicon",
         "fclose",
+        "feedback",
```
Comment (kmiya) [inline cspell/.cspell.json:300]:
```suggestion
```
`feedback` has been already included in the default en_US dictionary.

File: cspell/.cspell.json [hunk 3]
```diff
@@ -1039,6 +1040,7 @@
         "xorriso",
         "xosc",
         "XYZI",
+        "XYZIRADT",
```
Comment (kmiya) [inline cspell/.cspell.json:1043]: Question: What does `XYZIRADT` stand for? I'd like to add the meaning of the word to the commit message, like #87.

Comment (h-ohta) [reply cspell/.cspell.json:1043]: `XYZIRADT` is the message type, similar to `XYZI`.
It stands for X, Y, Z, Intensity, Ring, Azimuth, Distance and Timesramp.

Comment (h-ohta) [reply cspell/.cspell.json:1043]: I`ll add it to PR comment.

File: cspell/.cspell.json (hunk 2 above)
Comment (h-ohta) [reply cspell/.cspell.json:300]: Could you tell me the place of the en_US dictionary ? I also confirm it.

Comment (h-ohta) [reply cspell/.cspell.json:300]: Is this?
https://raw.githubusercontent.com/streetsidesoftware/cspell-dicts/main/dictionaries/cpp/cpp.txt
I can find `feedback`.
So I delete it.

**Comment by h-ohta** [comment]
I rebased

**Review by kmiya — APPROVED** [review]
Thanks!

**Comment by kmiya** [comment]
Oh, explanations of words are not included in the merge message. I will revert later.

**Comment by h-ohta** [comment]
Sorry, I forget it...

**Pull request merged**